### PR TITLE
add Sun stone to Evo stones

### LINKED ISSUE #350: Sun Stone missing from Evo Stone UI
The sun stone is missing from the evo stone ui, but will be needed in gen 2 to evolve Gloom (Nr.044) into Bellossom (Nr.182)

### PR DESCRIPTION
**Pull request by Quindon**
Added Sunstone to Items / Evo Stones modal 

Added small picture (32x32px) to asstes/images/items

closes #350

### CODE REVIEW

**Comment by Ishadijcks** [comment]
Can you fix your formatting please?

**Comment by Quindon** [comment]
Sorry, looked fine in notepad. Will change as soon as I'm home

File: src/components/itemModal.html [hunk 1]
```diff
@@ -53,6 +53,10 @@
                                         <img src="assets/images/items/Trade_stone.png">
                                         <div data-bind="text: player._itemList['Trade_stone']()"></div>
                                     </div>
+									<div class=col style="cursor: pointer" data-bind="css:{ 'item-selected': ItemHandler.stoneSelected() === 'Sun_stone'}, click: function(){ ItemHandler.stoneSelected('Sun_stone'); ItemHandler.pokemonSelected('')}">
```
Comment (RedSparr0w) [inline src/components/itemModal.html]:
```suggestion
                                    <div class="col" style="cursor: pointer" data-bind="css:{ 'item-selected': ItemHandler.stoneSelected() === 'Sun_stone'}, click: function(){ ItemHandler.stoneSelected('Sun_stone'); ItemHandler.pokemonSelected('')}">
```

File: src/components/itemModal.html [hunk 2]
```diff
@@ -53,6 +53,10 @@
...
+									<img src=assets/images/items/Sun_stone_small.png>
```
Comment (RedSparr0w) [inline src/components/itemModal.html]: Could you put the source in quote marks please
```suggestion
                                        <img src="assets/images/items/Sun_stone_small.png">
```

File: src/components/itemModal.html [hunk 3]
```diff
@@ -53,6 +53,10 @@
...
+									<div data-bind="text: player._itemList['Sun_stone']()"/>
```
Comment (RedSparr0w) [inline src/components/itemModal.html]: Could you please close this div, unfortunately [div elements do not allow self closing](https://stackoverflow.com/questions/7971716/is-it-ok-to-use-a-self-closing-div-tag).
```suggestion
                                        <div data-bind="text: player._itemList['Sun_stone']()"></div>
```

File: src/components/itemModal.html (hunk 2 above)
Comment (Quindon) [reply src/components/itemModal.html]: Sun_Stone is a 90x90px asset used in the underground, which is way to big for evo stone display 
Sun_Stone_Small is the same asset, only 32x32px

 tag is not closed anywhere else in the evo display and acc. [stackoverflow](https://stackoverflow.com/questions/23890716/why-is-the-img-tag-not-closed-in-html/23890817) doesent need to be

Comment (RedSparr0w) [reply src/components/itemModal.html]: Ah sorry, missed that a file had been uploaded with the PR :sweat_smile: 

Could you just add the quote marks around the `src` for the `img` and the `class` in the `div` above, and close the `div` below.

**Pull request merged**